website/docs/user-guide/concepts/external-services.md

```diff
@@ -0,0 +1,102 @@
+# External Services
+
+This document describes how to provide external services to ORT runs, like for instance source code or artifact repositories.
+
+## Infrastructure Services
+
+External services are always accessed via _Infrastructure Services_. An Infrastructure Service can be created via API/UI or defined in a project's `.ort.env.yml`. It has the following properties:
```

Comment:
It's a good practice to always start a sentence on a new line to minimize the diff when text is changed.